tests/articles.test.js

```diff
@@ -0,0 +1,49 @@
+import '@babel/polyfill';
+import chai from 'chai';
+import chaihttp from 'chai-http';
+import server from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+import errorMessage from '../src/helpers/constants/error.messages';
+
+chai.use(chaihttp);
+chai.should();
+
+describe('GET /articles/:slug', () => {
+  it('Should return the article', (done) => {
+    chai
+      .request(server)
+      .get('/api/articles/How-to-create-sequalize-seeds')
+      .end((err, res) => {
+        res.should.have.a.status(status.OK);
```

Comment:
@truestbyheart let's confirm that the expected article is returned. Check it's slug, title, description etc here, not just that an object with property article is returned.

Reply:
Alright @MemunaHaruna let me add something to check this properties